practice-one/src/index.html

```diff
@@ -3,44 +3,82 @@
-                <li>
-                    <a class="text-primary navbar-item text-dark " href="#">PLACES</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PRODUCTS</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PROJECTS</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PORTFOLIO</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">ABOUT US</a>
-                </li>
-            </ul>
-        </nav>
-    </header>
-    <!-- End Header -->
+
+    <!-- Start Footer -->
+    <footer class="footer-background">
+        <div class="footer-content container">
+            <div class="footer-list">
+                <div class="footer-item">
+                    <h4 class="item-title">contact us</h4>
+                    <address class="address">
+                        <p class="description">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div>
+                                <a class="description" href="tel:+2357938917938">+235 793-891-7938</a>
+                            </div>
+                            <div>
+                                <a class="description" href="mailto:contact@hofmannui.net">contact@hofmannui.net</a>
+                            </div>
+                        </div>
+                    </address>
+                </div>
+                <div class="footer-item">
+                    <h4 class="item-title">follow us</h4>
+                    <div class="socials">
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/Pinterest.svg" alt="Pinterest"/>
+                        </a>
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/twitter.svg" alt="Twitter"/>
+                        </a>
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/facebook.svg" alt="Facebook"/>
+                        </a>
+                    </div>
+                </div>
+                <div class="footer-item">
+                    <h4 class="item-title">about us</h4>
+                    <p class="description">SmartWeb was made to help you quickly create beautiful<br>layouts of responsive websites.</p>
+                </div>  
+            </div>
+
+            <hr class="footer-separator">
+
+            <div class="footer-list">
+                <div class="description">© 2023 SmartWeb</div>
+                <nav>
+                    <ul class="menu">
+                        <li>
+                            <a class="menu-item" href="#">home</a>                    
```

Comment:
@HoangLiarPhuongLe The same:
`Please use href="javascript:void(0)" instead of href="#" so that the link will have no associated action. Clicking on it will not cause the page to reload or redirect to another location.`

Reply:
I fixed it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/8c970229-4c71-43f8-8f36-cbcc71c5eb01)